manager/integration/tests/common.py

```diff
@@ -194,6 +194,8 @@ def create_backup(client, volname, data={}, labels={}):
     # have added extra Labels (for things like BaseImage).
     for key, val in labels.iteritems():
         assert new_b["labels"].get(key) == val
+
+    volume = wait_for_backup_completion(client, volname, snap["name"])
```

Comment:
I think we should have code removed because of adding this function?

Reply:
I checked all the places where create_backup is called, and there is no redundant logic except the place we talked before.